dev/tasks/tasks.yml

```diff
@@ -911,6 +912,13 @@ tasks:
       - Apache.Arrow.{no_rc_version}.nupkg
       - Apache.Arrow.{no_rc_version}.snupkg
 
+  ######################## R packages & binaries ##############################
+  r-nightly-packages:
+    ci: github
+    template: r/github.nightly.yml
```

Comment:
Do we intend to upload the produced binaries to crossbow similarly to other packaging tasks? If yes, then you need to add the list of expected artifacts. Example from a wheel task:

```yaml
    artifacts:
      - pyarrow-{no_rc_version}-cp38-cp38-macosx_11_0_arm64.whl
```

Reply:
I don't think uploading the artifacts to the releases is necessary as they are available in the job summary and those should only be needed on PR runs to check/test so we don't need them long term. The actual nightlies will be uploaded to the apache server and available as long as we choose (14 versions for each atm).

Reply:
We will want to upload release artifacts, though there's more work to be done to add this to the release process, I suspect.